docs/news/news-v5.0.x.rst

```diff
@@ -0,0 +1,211 @@
+Open MPI v5.0.x series
+======================
+
+This file contains all the NEWS updates for the Open MPI v5.0.x
+series, in reverse chronological order.
+
+Open MPI version 5.0.0rc4
+-------------------------
+:Date: 30 March 2022
+
+.. caution:: MPIR API has been removed
+
+   As was announced in summer 2017, Open MPI has removed support of
+   MPIR-based tools beginning with the release of Open MPI v5.0.0.
+
+   The new PRRTE based runtime environment supports PMIx-tools API
+   instead of the legacy MPIR API for debugging parallel jobs.
+
+   See https://github.com/openpmix/mpir-to-pmix-guide for more
+   information.
+
+.. caution:: zlib is suggested for better user experience
+
+   PMIx will optionally use zlib to compress large data streams.
+   This may result in faster startup times and
+   smaller memory footprints (compared to not using compression).
+   The Open MPI community recommends building zlib support with PMIx,
+   regardless of whether you are using an externally-installed PMIx or
+   the PMIx that is installed with Open MPI.
+
+- Updated to use PMIx ``v4.2`` branch - current hash: ``1b86a35``.
+- Updated to use PRRTE ``v2.1`` branch - current hash: ``91f791e``.
+
+- New MPI-4.0 features:
```

Comment:
Before we release, we should have a disclaimer here stating that while Open MPI v5.0.0 is _NOT_ yet MPI-4.0 compliant, we do have a number of MPI-4.0 features implemented for developers to begin to use.